velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -1215,3 +1208,24 @@ TEST_F(ParquetReaderTest, testLzoDataPage) {
           .str(),
       "31232");
 }
+
+TEST_F(ParquetReaderTest, testEmptyDataPage) {
+  const std::string sample(getExampleFilePath("snappy.parquet"));
```

Comment:
Maybe rename this file to empty_v2datapage.parquet?

Reply:
Fixed!